src/parser/FileParser.kt

```diff
@@ -0,0 +1,108 @@
+package parser
+
+import CoffeeShop
+import Coordinate
+import java.io.File
+
+/**
+ * Defines a parser.FileParser object, containing different operations on the content of a given .csv file
+ *
+ * created by AlexandraV on 12/25/2018
+ */
+data class FileParser(var fileName: String, var userCoordinate: Coordinate) {
+    val coffeeShopsList: MutableList<CoffeeShop> = mutableListOf()
+    private val lines: List<String>? = File(fileName).useLines { it.toList() } // the reader is closed automatically
```

Comment:
What about using a delegate here like [lazy](https://kotlinlang.org/docs/reference/delegated-properties.html#lazy) do you think that will be appropriate?